dietpi/dietpi-software

```diff
@@ -12906,7 +12906,7 @@ Documentation=https://paper.readthedocs.io/
 User=papermc
 SyslogIdentifier=papermc
 WorkingDirectory=/mnt/dietpi_userdata/papermc
-ExecStart=$(command -v java) -jar /opt/papermc/paperclip.jar --nogui --noconsole
+ExecStart=$(command -v java) -Xmx1024m -jar /opt/papermc/paperclip.jar --nogui --noconsole
```

Comment:
bit offtopic I guess, I have no idea what any of this is, but these arguments here should be editable by the user, as its something often used for performance tweaking. especially on low power devices like a pi, this is important.

Reply:
it can be changed by adjusting the service file. A feature that is available via `dietpi-services`. There you would need to adjust `ExecStart` command. I doubt there will be a GUI for setting this values.

Reply:
Good point. We could add it to our docs and as well link to the available command arguments from here.

nogui and noconsole are basically required to run it reasonably in a systemd unit, for other things we could as well add an environment file like `/etc/default/papermc` as well, but when explaining the method via dietpi-services good, it is probably no big benefit.